lib/jobs/run-sku-graph.js

```diff
@@ -41,7 +41,7 @@ function runSkuGraphJobFactory(
         RunSkuGraphJob.super_.call(this, logger, options, context, taskId);
 
         this.nodeId = this.options.nodeId;
-        assert.isMongoId(this.nodeId);
+        assert.string(this.nodeId);
```

Comment:
I like that. Would be trivial since we already monkey-patch assert https://github.com/RackHD/on-core/blob/master/lib/common/assert.js